service/controller/key/key.go

```diff
@@ -72,6 +73,10 @@ func DNSIP(clusterIPRange string) (string, error) {
 	return ip.String(), nil
 }
 
+func IsBundle(appName string) bool {
+	return strings.HasSuffix(appName, "-bundle")
```

Comment:
How does that work for `security-pack`, for example?

Reply:
It does not and it should be renamed to security-bundle https://intranet.giantswarm.io/docs/product/pdr/008_app_bundle_naming/